### ISSUE TITLE
`spaced-comment` rule warns line-endings of Windows.

### ISSUE THREAD

**Issue by mysticatea** [eslint/eslint#3005]
This is related with #2884.

On the issue, the problem around CRLF seems to be not fixed completely.
I tried with `master` branch, this rule still warn at normal block comments.

## test.js

```js
/* eslint spaced-comment:2 */

/*
 * test
 */

/**
 * test
 */
```

## Command & Actual

```
> git log -1
commit e88f8d816072957fcf9c67188ee609f3d40a4957
...
> node .\bin\eslint.js test.js --no-eslintrc

test.js
  3:1  error  Expected space or tab after /* in comment  spaced-comment

✖ 1 problem (1 error, 0 warnings)
```

## Expected

No errors.

----

To fix, I guess `&& node.value.indexOf("\r") !== 0` is added into [here](https://github.com/eslint/eslint/blob/e88f8d816072957fcf9c67188ee609f3d40a4957/lib/rules/spaced-comment.js#L86) merely.
